workshops/maze/README.md

```diff
@@ -0,0 +1,113 @@
+# Friendly Maze Game
+
+## 1. Demo
+
+Your mission is to build your own version of
+[The Scary Maze Game](http://www.playscarymazegame.net/play-scary-maze-game/).
+
+Navigate your mouse through the maze without touching the walls:
+
+![](img/demo.gif)
+
+<a href="http://output.jsbin.com/xivifu/12" target="_blank_">![](img/bttn_play_now.png)</a>
+<a href="http://jsbin.com/xivifu/12/edit?js,output" target="_blank_">![](img/bttn_view_code.png)</a>
+
+## 2. How to Use This Tutorial
+
+In this tutorial, instead of walking you through how to build everything from
+scratch, we give you the code for how to make the maze game show above and
+your mission will be to make your own version of it.
+
+This workshop also, assumes that you have done the
+[Dodge](https://github.com/hackedu/hackedu/blob/master/workshops/dodge/README.md)
+workshop. Make sure you do that workshop first.
+
+## 3. Setup
+
+1. If you're more experienced and want to use your own editor, go ahead. Use the
```

Comment:
`Use the below files to your editor` doesn't sound that great. `Save the following files to your computer` or `Open the following files in your editor of choice` sounds better